.vscode/settings.json

```diff
@@ -0,0 +1,69 @@
+{
+    "files.associations": {
```

Comment:
Is all of this needed? Seems odd we'd need to specify all these file types to associate to cpp.

Reply:
I don't think this is needed as it pertains to the IDE, not the project itself. This has been removed and the unit tests are still passing.